askama/src/filters/humansize.rs

```diff
@@ -43,82 +41,190 @@ impl FastWritable for FilesizeFormatFilter {
         dest: &mut W,
         values: &dyn Values,
     ) -> crate::Result<()> {
-        if self.0 < 1e3 {
+        if self.0 < 1000 {
```

Comment:
Since it's only `if else`, could you return in the `if` please? Like that the whole `else` block can be unindented by one level.